products/opensuse/main.pm

```diff
@@ -38,6 +38,10 @@ sub is_leap {
     return get_var('VERSION') =~ /^42:S/;
 }
 
+sub is_leap_42_3_or_newer {
+    return is_leap && check_var('VERSION', '42.3');
+}
+
```

Comment:
What I see is a match for Leap 42.3, not "Leap 42.3 or newer". Or I am missing something?